src/components/app.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import SearchForm from './search-form';
+import '../styles/app.scss';
+import axios from 'axios';
+
+const url = 'https://mcr-codes-weather.herokuapp.com/forecast?city=';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
```

Comment:
I can't find where you are setting the initial state of the city property. I assume you're relying on the fact that the api defaults to returning the data for Manchester if no city is specified. this works with the api we've given you but probably wouldn't with a lot of other ones. For now I would probably say to set the initial state to `'manchester'` to avoid any errors that could pop up.